riot.d.ts

```diff
@@ -42,7 +42,13 @@ export type AutobindObjectMethods<Object, Component extends RiotComponent> = {
     : Object[K]
 }
 
-export interface RiotComponent<Props = any, State = any> {
+export type DefaultProps = Record<string, any>;
```

Comment:
Please use `PropertyKey` here as well thanks

Reply:
I used string because the properties in props comes from the tag attributes, so wouldn’t they be always a string?

Reply:
You can pass props as follows `<my-component {...{ [Symbol('foo')]: 'foo', 2: 'number', 'string': 'a string' }}>`